public/scss/base/_flow.scss

```diff
@@ -0,0 +1,27 @@
+.flow > * + * {
+  margin-block-start: var(--flow-space, 1em);
+}
```

Comment:
This is based off https://andy-bell.co.uk/my-favourite-3-lines-of-css/ and seems really handy for handling space between elements.